web/src/composebox_typeahead.ts

```diff
@@ -373,6 +373,10 @@ function handle_keydown(
                 handle_enter($("textarea#compose-textarea"), e);
             }
         } else if (on_topic || on_pm) {
+            // Prevent form submission when pressing Enter in the topic or recipient box
+            if (keydown_util.is_enter_event(e)) {
+                e.preventDefault(); // Prevent default Enter behavior
+            }
```

Comment:
Can you explain why you believe this will work? How did you test it?

Reading this code, it seems to me like it shouldn't be possible for it to work.

Reply:
I'd also have thought that perhaps it'd be better to place this logic where we send a message on enter, not here, but that's more about readability.

Reply:
With ### stopPropagation, it wouldn't

Reply:
`stopPropagation()` wouldn't stop the default behaviour of the form so I had to use `preventDefault()`, as for test I woud upadate on it, and  we can put the logic where we send message.